### ISSUE TITLE
Inactive label remains when tab becomes active

### ISSUE THREAD

**Issue by emadd** [uias/Tabman#115]
![simulator screen shot jul 11 2017 8 14 35 am](https://user-images.githubusercontent.com/240649/28067897-295d95a6-6611-11e7-9e16-e1e29d438ab9.png)

**Comment by msaps** [comment]
Hi,

Is your indicator color translucent by any chance (`appearance.indicator.color`)? Currently the underlying (non-highlighted) buttons are not masked by the indicator which is why it might be visible. So this is something that definitely needs looking at! 

Also, looks as though you are using a custom font? From what I can see I don't think this is being correctly applied to the selected buttons. 

Would you be able to provide your `TabmanBar.Appearance` configuration? Just so I can accurately reproduce and see what's going on ? ☺️

Thanks!

**Comment by emadd** [comment]
No alpha at play as far as I can tell.  The font appears to enlarge slightly when selected, but the unselected state label is obviously still there.

```
let darkTabbedColor: UIColor = UIColor.init(hexString: "#1c1f22")
let lightTabbedColor: UIColor = UIColor.init(hexString: "#303336")

self.bar.appearance = TabmanBar.Appearance({ (appearance) in
            appearance.style.showEdgeFade = true
            appearance.state.color = UIColor.white.darken(byPercentage: 0.3)
            appearance.state.selectedColor = .white
            appearance.style.background = TabmanBarBackgroundView.BackgroundStyle.solid(color: lightTabbedColor)
            appearance.indicator.color = darkTabbedColor
            appearance.layout.itemVerticalPadding = 0
            appearance.indicator.bounces = true
            appearance.indicator.lineWeight = .normal
            appearance.layout.height = TabmanBar.Height.explicit(value: 37.0)
            appearance.text.font = UIFont(name: "OpenSans", size: 15)
            appearance.layout.edgeInset = 0
            appearance.layout.interItemSpacing = 1
            
})
```

**Comment by msaps** [comment]
This is now resolved as of [0.7.3](https://github.com/uias/Tabman/releases/tag/0.7.3). 🎉
